resharper/src/resharper-unity/Rider/UnityHost.cs

```diff
@@ -0,0 +1,17 @@
+﻿using JetBrains.ProjectModel;
+using JetBrains.ReSharper.Host.Features;
+using JetBrains.Rider.Model;
+
+namespace JetBrains.ReSharper.Plugins.Unity.Rider
+{
+    [SolutionComponent]
+    public class UnityHost
+    {
+        public RdUnityModel Model { get; }
+
+        public UnityHost(ISolution solution)
+        {
+            Model = solution.GetProtocolSolution().GetRdUnityModel();
```

Comment:
Why do we need such a component? The clients could get access to model with smth like
solution.GetUnityModel(), where GetUnityModel() would be an extension method.

Reply:
It is much more sustainable in a long way when we'll have to add some Unity-specific not related to the protocol. Plus injecting the component seems much more convenient to me than `solution.GetProtocolSolution().GetRdUnityModel();` in each constructor. Encapsulating the protocol just seems more readable to me, I guess, and we have a lot of them in RIder.